src/init.cpp

```diff
@@ -558,7 +558,7 @@ bool AppInit2()
     
     fUseFastIndex = GetBoolArg("-fastindex", false);
 
-    nMinerSleep = GetArg("-minersleep", 500);
+    nMinerSleep = GetArg("-minersleep", 8000);
```

Comment:
Not a single second. There is specific period of 16 seconds where staking is allowed. The (bit inverse of) stake mask is applied to timestamp effectively eliminating the difference when of staking anywhere during that 16 seconds. It was introduced to save stakers from having to check every second.